### ISSUE TITLE
Cannot search using short Report ID

### ISSUE THREAD

**Issue by m-natarajan** [Expensify/App#49822]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 9.0.40-1
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** @heyjennahay 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1727351437300099

## Action Performed:

1. Go to Expenses > Filters > Report ID
2. Enter long report ID (2462507380916573)   (Actual report iD R00BHFp32GTN (Long ID: 2462507380916573))
3. Click Save > View results
4. Expenses are displayed
5. Return to Filters and enter the same report's short report ID(R00BHFp32GTN)  (Actual report iD  (Long ID: 2462507380916573))
6. Click Save > View results

## Expected Result:
Expense should be displayed
## Actual Result:
No expenses are displayed

## Workaround:
unknown
## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android: Native
- [x] Android: mWeb Chrome
- [x] iOS: Native
- [x] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [x] MacOS: Desktop

## Screenshots/Videos
![image (7)](https://github.com/user-attachments/assets/33e40b1f-b3ef-474a-8413-efe3ea7ff50e)
![image (8)](https://github.com/user-attachments/assets/5fd5ceb9-277b-4185-8768-c39dcdc67bda)
![image (9)](https://github.com/user-attachments/assets/e85f9dd2-a495-4d4e-98f6-8a888945fadb)

https://github.com/user-attachments/assets/04577da9-528f-4fd1-8e75-064b9c4522a8

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by luacmartins** [comment]
@JmillsExpensify coming from your comment, we need to add some hint so users are aware that they can include both long and short reportIDs

**Comment by luacmartins** [comment]
I'll work on this tomorrow

**Comment by luacmartins** [comment]
PR in review

**Comment by luacmartins** [comment]
PR merged, pending deploy

**Comment by JmillsExpensify** [comment]
On the hint text, I think I'm starting to doubt that this is needed. Curious for your thoughts @jamesdeanexpensify? Basically we have two types of reportIDs, both short and long. LongIDs is what we use in NewDot URLs, though in CSV/Excel exports we use the shortID due to limitations of Excel. We are adding support for both forms of reportIDs, so arguably we don't need to say anything given that either reportID entered will be supported.

**Comment by heyjennahay** [comment]
Noting we also use short ID on customer's bank statements for reimbursements

**Comment by luacmartins** [comment]
>We are adding support for both forms of reportIDs, so arguably we don't need to say anything given that either reportID entered will be supported.

I agree with this. Given that both are supported, I think having the field with the `reportID` label is enough.

**Comment by luacmartins** [comment]
@JmillsExpensify @jamesdeanexpensify I'm gonna close this issue, please reopen if you think we should add the hint

**Comment by jamesdeanexpensify** [comment]
I like keeping it as simple/minimal as possible, too. If we support all types of reportIDs, then I agree we don't need the hint text to clarify more!
